packages/api/lib/semaphoreUtils.js

```diff
@@ -1,7 +1,7 @@
 const { dynamodbDocClient } = require('@cumulus/aws-client/services');
```

Comment:
From my reading of the style guide, this should be called `SemaphoreUtils.js`:

* https://github.com/airbnb/javascript/blob/master/README.md#naming--filename-matches-export
* https://github.com/airbnb/javascript/blob/master/README.md#naming--PascalCase-singleton